### ISSUE TITLE
Question about training behaviors on CASIA

### ISSUE THREAD

**Issue by lymanblue** [ZhaoJ9014/face.evoLVe.PyTorch#89]
Hi~

From the original ArcFace paper, the IR_50 model trained on CASIA can achieve 99.5% accuracy on LFW in Table 2.
However, we are not able to reproduce the result from this PyTorch implementation.
Have you successfully reproduced the results in ArcFace paper?
If so, could you share the hyper-parameters?

The training loss is usually large for training on CASIA (similar to issue 52).
Could you share your training experiences or logs for training on CASIA?
Because the model zoo shows the training results for the training data MS1M instead of CASIA.

Thank you.

**Comment by ZhaoJ9014** [comment]
> Hi~
> 
> From the original ArcFace paper, the IR_50 model trained on CASIA can achieve 99.5% accuracy on LFW in Table 2.
> However, we are not able to reproduce the result from this PyTorch implementation.
> Have you successfully reproduced the results in ArcFace paper?
> If so, could you share the hyper-parameters?
> 
> The training loss is usually large for training on CASIA (similar to issue 52).
> Could you share your training experiences or logs for training on CASIA?
> Because the model zoo shows the training results for the training data MS1M instead of CASIA.
> 
> Thank you.

Hi, thank you for your question. We did not train the models on CASIA since MS1M is a larger-scale dataset for learning face recognition models, and we do achieved promising performance on several evaluation benchmark datasets, as can be seen from model zoo in our repo :)
